finder-app/makefile

```diff
@@ -0,0 +1,11 @@
+# Makefile to allow for cross_compilation on ARM64 target as well as native Linux
+
+CROSS_COMPILE = 
+cc = gcc
```

Comment:
@GautamaGandhi I think you'd typically use a capital `CC` here.

Reply:
@dwalkes thank you for the feedback. Is using capital case the normal convention when writing variables in the makefile? I shall follow this.

Reply:
Yes, and this particular variable is capital `CC` - see https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html